src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.td

```diff
@@ -201,7 +212,23 @@ def replaceONNXDivPattern : Pat<
   (ONNXDivOp $x, $y),
   (ZHighUnstickOp (ZHighDivOp (ZHighStickOp:$s_x $x, (NoneLayoutAttr)),
                               (ZHighStickOp $y, (NoneLayoutAttr)),
-                              (returnType $s_x)))
+                              (returnType $s_x))),
+  [], [],
+  (addBenefit 0)
+>;
```

Comment:
The condition of the pattern is implicitly mixed with the condition in LegalityCheck. Should we check the condition here redundantly, or any better approach?

Reply:
By redundancy, I meant to add the condition checked by LegalityCheck into the pattern so that the pattern can stand alone.